install.rb

```diff
@@ -294,6 +308,12 @@ def prepare_installation
     bindir = RbConfig::CONFIG['bindir']
   end
 
+  if not InstallOptions.localedir.nil?
+    localedir = InstallOptions.localedir
+  else
+    localedir = "/opt/puppetlabs/puppet/share/puppet/locale"
```

Comment:
We should add a Windows path. However, we don't seem to have an analog yet to the "share" folder used in the Unix path. Does anyone have a suggestion for where it might make sense to put these files, given Windows file structure? @Iristyle or @glennsarti ?